File: sample-app/IdentityManagement/Entities/CompanyInfo.cs
using IdentityManagement.Utilities;
using System.ComponentModel.DataAnnotations;

namespace IdentityManagement.Entities
{
    public class CompanyInfo
    {
        public string Id { get; set; }
        [Required]
        [Display(Name = "User Name")]
        public string UserName { get; set; }
        [Required]
        public string Email { get; set; }
        [DataType(DataType.Password)]
        public string Password { get; set; }
        public EnumUserStatus Status { get; set; }
        [Required]
        [Display(Name = "Company Name")]
        public string Name { get; set; }
        [Required]
        public string City { get; set; }
        [Required]
        public string State { get; set; }
        public bool IsActive { get; set; }
    }
}

File: sample-app-java/src/main/java/com/opsgility/contosojobs/ContosoJobsApplication/Entity/CompanyInfo.java
package com.opsgility.contosojobs.ContosoJobsApplication.Entity;

import javax.validation.constraints.NotNull;

public class CompanyInfo {
	private String id;
	@NotNull(message="required")
	private String userName;
	
	@NotNull(message="required")
	private String email;
	
	@NotNull(message="required")
	private String password;
	
	@NotNull(message="required")
	private String name;
	
	@NotNull(message="required")
	private String city;
	
	@NotNull(message="required")
	private String state;
	
	public CompanyInfo() {
		
	}

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getUserName() {
		return userName;
	}

	public void setUserName(String userName) {
		this.userName = userName;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getCity() {
		return city;
	}

	public void setCity(String city) {
		this.city = city;
	}

	public String getState() {
		return state;
	}

	public void setState(String state) {
		this.state = state;
	}
	
}
